LiveIngest/LiveEventWithDVR/Program.cs

```diff
@@ -77,7 +77,7 @@ private static async Task RunAsync(ConfigWrapper config)
             string drvStreamingLocatorName = "streamingLocator" + uniqueness;
             string archiveStreamingLocatorName = "fullLocator-" + uniqueness;
             string drvAssetFilterName = "filter-" + uniqueness;
-            string streamingEndpointName = "se";  // Change this to your Endpoint name.
+            string streamingEndpointName = "default";  // Change this to your Endpoint name.
```

Comment:
I cant recall why we used "se" in the past here and not "default". Good idea to change it back to default now.